framework/decode/metadata_json_consumer.h

```diff
@@ -314,6 +315,40 @@ class MetadataJsonConsumer : public Base
         WriteBlockEnd();
     }
 
+    virtual void ProcessBuildVulkanAccelerationStructuresMetaCommand(
+        format::HandleId                                                           device_id,
+        uint32_t                                                                   info_count,
+        StructPointerDecoder<Decoded_VkAccelerationStructureBuildGeometryInfoKHR>* geometry_infos,
+        StructPointerDecoder<Decoded_VkAccelerationStructureBuildRangeInfoKHR*>*   range_infos) override
+    {
+        auto&              jdata        = WriteMetaCommandStart("BuildVulkanAccelerationStructuresMetaCommand");
```

Comment:
To be consistent with the rest of the metadata JSON this should be the name from format.h, "VulkanBuildAccelerationStructuresCommand"